File: app/src/main/java/com/kuopin/frame/net/NetProvider.java
package com.kuopin.frame.net;

import com.benjamin.http.RetrofitManager;

/**
 * @author Ben
 * @date 2019/3/19
 */
public class NetProvider {

    public static APIService getInstance() {
        return RetrofitManager.create(APIService.class);
    }

    public static APIService getInstance(String baseUrl) {
        return RetrofitManager.create(APIService.class, baseUrl);
    }
}

File: app/src/main/java/com/kuopin/frame/net/NetProvider.kt
package com.kuopin.frame.net

import com.benjamin.http.RetrofitManager

/**
 * @author Ben
 * @date 2019/3/19
 */
object NetProvider {

    val instance: APIService
        get() = RetrofitManager.create(APIService::class.java)

    fun getInstance(baseUrl: String): APIService {
        return RetrofitManager.create(APIService::class.java, baseUrl)
    }
}
